Shift3/boilerplate-client-react
# chore: remove wiki deployment from ci jobs

## Changes
1. Removed `deploy_wiki.sh` script
2. Removed CircleCI jobs that deployed the wiki
3. Removedk `wiki` folder with markdown pages

## Purpose
We don't really use the wiki or make big changes to the wiki that require commit history.

## Testing
1. Pull in the changes to your local copy of this branch and restart app.
2. Check that the app still runs and works as expected. There shouldn't be any functional changes to the app.
